HHS/Head-Start-TTADP
# [Prod] TTAHUB-173, 325, 486, 498

## Description of change

* TTAHUB-173: Add Report List widget to Dashboard page.

* TTAHUB-325: Add Program Specialist to CSV export.

* TTAHUB-486: Add report filter for grant Number (BE only).

* TTAHUB-498: Remove Program Type from AR form.

* TTAHUB-484: Add report filter for program type (BE only)

* TTAHUB-514: Grantee record updates.

* TTAHUB-346: Pull in grantee state (i.e. 'KS') from HSES

* TTAHUB-496: Add script to transform legacy target population data

## How to test

* TTAHUB-173: View the Regional Dashboard page. The bottom of the page should now show a Report List widget that reflects the filters applied to the overall page. All functionality of the widget should behave the same as the landing page (paging, exporting, sorting).

* TTAHUB-325: Export the AR csv. There should now be a column called 'Program specialist'. This column should show all program specialists for the AR separated by a new line.

* TTAHUB-486: No test needed at this time back end change ONLY.
* TTAHUB-484: No test needed at this time back end change ONLY.

* TTAHUB-498: Creating a new AR should no longer have an entry for program type. The grantee selection should now show all program types as a comma separated list after the grant number. Viewing an AR should also no longer have its own program type section (with the exception of legacy). Exporting the CSV should also show a column for Program types separated by new lines. 

* TTAHUB-514: On the Grantee Record page. Hovering over a grant number should show redirects to HSES text. Grant list order shows Active grants first. Filter shows '- Select -' if no Topic or Condition has been selected.

* TTAHUB-496: Script has been run on sandbox.

* TTAHUB-346: No test needed at this time back end change ONLY.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-173
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-325
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-486
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-498
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-514
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-346
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-496

## Checklists

### Every PR


~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy


### After merge/deploy
